tutorials/circuits/2_plotting_data_in_qiskit.ipynb

```diff
@@ -7,715 +7,42 @@
-    "               color=['orange', 'black'], bar_labels=False)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Using the output from plot_histogram()\n",
-    "\n",
-    "When using the plot_histogram() function it returns a `matplotlib.Figure` for the rendered visualization. Jupyter notebooks understand this return type and render it for us in this tutorial, but when running outside of Jupyter you do not have this feature automatically. However, the `matplotlib.Figure` class natively has methods to both display and save the visualization. You can call `.show()` on the returned object from `plot_histogram()` to open the image in a new window (assuming your configured matplotlib backend is interactive). Or alternatively you can call `.savefig('out.png')` to save the figure to `out.png`. The `savefig()` method takes a path so you can adjust the location and filename where you're saving the output."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Plot State <a name='state'></a>"
-   ]
-  },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In many situations you want to see the state of a quantum computer. This could be for debugging. Here we assume you have this state (either from simulation or state tomography) and the goal is to visualize the quantum state. This requires exponential resources, so we advise to only view the state of small quantum systems. There are several functions for generating different types of visualization of a quantum state\n",
-    "\n",
-    "```\n",
-    "plot_state_city(quantum_state)\n",
-    "plot_state_qsphere(quantum_state)\n",
-    "plot_state_paulivec(quantum_state)\n",
-    "plot_state_hinton(quantum_state)\n",
-    "plot_bloch_multivector(quantum_state)\n",
-    "```\n",
-    "\n",
-    "A quantum state is either a state matrix $\\rho$ (Hermitian matrix) or statevector $|\\psi\\rangle$ (complex vector). The state matrix is related to the statevector by \n",
-    "\n",
-    "$$\\rho = |\\psi\\rangle\\langle \\psi|,$$\n",
-    "\n",
-    "and is more general as it can represent mixed states (positive sum of statevectors) \n",
+    "<div class=\"alert alert-warning\">\n",
     "\n",
-    "$$\\rho = \\sum_k p_k |\\psi_k\\rangle\\langle \\psi_k |.$$\n",
+    "Deprecation Warning\n",
     "\n",
-    "The visualizations generated by the functions are:\n",
+    "As part of the Qiskit Documentation content restructuring efforts, the content of this tutorial has been moved to [qiskit.visualization](https://qiskit.org/documentation/apidoc/visualization.html) module API reference. You can find code examples in the API reference page of each function:\n",
     "\n",
-    "- `'plot_state_city'`: The standard view for quantum states where the real and imaginary (imag) parts of the state matrix are plotted like a city.\n",
+    "- plot_histogram\n",
```

Comment:
```suggestion
    "- `plot_histogram`",
```

Reply:
@1ucian0 Sorry for the long delay. I added the links. @SooluThomas Due to the [limitation](https://github.com/spatialaudio/nbsphinx/issues/301) of `nbsphinx`, backquotes and linking at the same time will not work. I choose linking over formatting. 

This is output if we add backquotes and links:
<img width="479" alt="image" src="https://user-images.githubusercontent.com/7631333/206209756-a014b9b4-84e4-47c6-b0ab-1ab6b1836492.png">

Reply:
Yes, let's choose linking over the code block.